cards/_components/Card/Embed.vue

```diff
@@ -30,13 +30,46 @@
 <script>
 export default {
   name: 'CardEmbed',
+
   computed: {
     embedCode() {
-      return `&#x3C;script&#x3E;(function(d,script){script=d.createElement(&#x27;script&#x27;);script.type=&#x27;text/javascript&#x27;;script.async=true;script.onload=function(){iFrameResize({log:true,checkOrigin:false})};script.src=&#x27;https://cdn.parlameter.si/v1/parlassets/js/iframeResizer.min.js&#x27;;d.getElementsByTagName(&#x27;head&#x27;)[0].appendChild(script);}(document));&#x3C;/script&#x3E;&#x3C;iframe frameborder=&#x22;0&#x22; width=&#x22;100%&#x22; src=&#x22;${this.url}&#x26;embed=true&#x22;&#x3E;&#x3C;/iframe&#x3E;`;
+      let newUrl = this.url;
+      if (!this.refresh) {
+        const todaysdate = new Date;
+        const today = '' + todaysdate.getDay() + '.' + todaysdate.getMonth() + '.' + todaysdate.getFullYear();
```

Comment:
Use template literals here. Also, I think we use [`date-fns`](https://date-fns.org/) to manipulate/format dates in other places, maybe take a look at that.